etc/kernel-launchers/kubernetes/scripts/launch_kubernetes.py

```diff
@@ -131,23 +152,44 @@ def launch_kubernetes_kernel(
                 #  print("{}".format(k8s_obj))  # useful for debug
                 pod_template = extend_pod_env(k8s_obj)
                 if pod_template_file is None:
-                    pod_created = client.CoreV1Api(client.ApiClient()).create_namespaced_pod(
-                        body=k8s_obj, namespace=kernel_namespace
-                    )
+                    try:
+                        pod_created = client.CoreV1Api(client.ApiClient()).create_namespaced_pod(
+                            body=k8s_obj, namespace=kernel_namespace
+                        )
+                    except ApiException as exc:
+                        if _parse_k8s_exception()(exc) == K8S_ALREADY_EXIST_REASON:
```

Comment:
I don't understand this syntax.  How are the _two_ sets of parenthesis handled?  I expected: `_parse_k8s_exception(exc)`.   

(I'm not up on all of the various python features.  When I tried searching for this syntax it appears to be used when embedded functions are involved, but that doesn't appear to be the case here so I'm curious.)

Reply:
Cool - glad it wasn't some other aspect of Python that I need to become familiar with! :smile: